ui/src/buckets/components/BucketsTab.tsx

```diff
@@ -121,15 +139,25 @@ class BucketsTab extends PureComponent<Props, State> {
             searchTerm={searchTerm}
             onSearch={this.handleFilterChange}
           />
-          <Button
-            text="Create Bucket"
-            icon={IconFont.Plus}
-            color={ComponentColor.Primary}
-            onClick={this.handleOpenModal}
-            testID="Create Bucket"
-            status={this.createButtonStatus}
-            titleText={this.createButtonTitleText}
-          />
+          <div className="buckets-buttons-wrap">
+            <FeatureFlag name="demodata">
```

Comment:
stylistically, you could  remove an extra layer of elements and logic by doing something like:

```ts
{isFlagEnabled('demodata') && demoDataBuckets.length > 0 &&
```
and getting rid of `<FeatureFlag name="demodata">`. but it's up to preference on whether one is better than the other i think.

Reply:
👍